src/componets/navbar/navbar.tsx

```diff
@@ -11,7 +11,7 @@ const navbar = () => {
           Plan a trip
         </Link>
         <Link to="/discover" className="mx-5">
-          Discover
+          Discover.
```

Comment:
Hey good thought ab out x, other solutions to tis are y

- you should check out [this doc](https://react.dev/reference/react/useId)

Reply:
Highly recommended / required: 
Recommended: You could swap x out with y, see some docs here
margin: 20px 30px 20px 30px
Nit: This could be refactored to just margin: 20px 30px.